common/query_manager/query_parser/ppl_query_parser.ts

```diff
@@ -20,6 +20,12 @@ export class PPLQueryParser {
 
   getStats() {
     this.visitor = new StatsAstBuilder();
-    return this.visitor.visitRoot(this.parser!.root()).getTokens();
+    let inter = null;
```

Comment:
'null' is the default value for this function, is used when there is no stats function to parse

Reply:
why is it called inter?

Reply:
I just chose it as shorthand for intermediate but I can change the name is there is something else we use that is more colloquial.